lib/llm/providers/openai.rb

```diff
@@ -44,5 +43,11 @@ def auth(req)
     def response_parser
       LLM::OpenAI::ResponseParser
     end
+
+    def transform_prompt(prompt)
```

Comment:
This method can return `nil`. We should return `prompt` if no other condition matches.

Reply:
@0x1eef completely overlooked that! for some reason i thought just by returning prompt in the provider superclass we should be good, but that's obviously not the case.